tests/chainer_tests/functions_tests/activation_tests/test_randomized_leaky_relu.py

```diff
@@ -0,0 +1,74 @@
+import random
+import unittest
+
+import numpy
+
+import chainer
+from chainer import cuda
+from chainer import functions
+from chainer import gradient_check
+from chainer import testing
+from chainer.testing import attr
+from chainer.testing import condition
+
+
+@testing.parameterize(*testing.product({
+    'shape': [(3, 2), (5, 6)],
+    'dtype': [numpy.float16, numpy.float32, numpy.float64],
+}))
+class TestRReLU(unittest.TestCase):
+
+    def setUp(self):
+        # Avoid unstability of numeraical grad
+        self.x = numpy.random.uniform(-1, 1, self.shape).astype(self.dtype)
+        for i in numpy.ndindex(self.shape):
+            if -0.05 < self.x[i] < 0.05:
+                self.x[i] = 0.5
```

Comment:
I refered test_clipped_relu.py, test_elu.py, test_leaky_relu.py,  test_prelu.py , test_relu.py and test_selu.py.
They say if values around zero, numerical gradient is not stable.